setup.py

```diff
@@ -21,6 +21,7 @@
     ],
     install_requires=[
         'requests',
+        'cartographer',
```

Comment:
might want to pin that to a version, newer cartographer versions require python 3.6 or asynq